packages/gatsby-theme/src/components/BlogHead/index.js

```diff
@@ -0,0 +1,16 @@
+import BlogHead from "./BlogHead"
+import BlogHeadImage from "./BlogHeadImage"
+import BlogHeadTitle from "./BlogHeadTitle"
+import BlogHeadAuthor from "./BlogHeadAuthor"
+import BlogHeadContainer from "./BlogHeadContainer"
+import BlogHeadDate from "./BlogHeadDate"
+import BlogHeadCredits from "./BlogHeadCredits"
+
+BlogHead.Image = BlogHeadImage
+BlogHead.Title = BlogHeadTitle
+BlogHead.Author = BlogHeadAuthor
+BlogHead.Date = BlogHeadDate
+BlogHead.Container = BlogHeadContainer
+BlogHead.Credits = BlogHeadCredits
+
+export default BlogHead
```

Comment:
Please add this component to the `ui` index under `frontend/packages/gatsby-theme-octahedroid/index.js`